bindinfo/handle.go

```diff
@@ -447,7 +447,10 @@ func (h *BindHandle) Size() int {
 }
 
 // GetBindRecord returns the BindRecord of the (normdOrigSQL,db) if BindRecord exist.
-func (h *BindHandle) GetBindRecord(hash, normdOrigSQL, db string) *BindRecord {
+func (h *BindHandle) GetBindRecord(hash, normdOrigSQL, db string, matchDB bool) *BindRecord {
+	if matchDB {
+		return h.bindInfo.Load().(cache).getBindRecordWithDB(hash, normdOrigSQL, db)
+	}
 	return h.bindInfo.Load().(cache).getBindRecord(hash, normdOrigSQL, db)
 }
```

Comment:
How about removing `matchDB` and implementing it as:
``` 
if record := h.bindInfo.Load().(cache).getBindRecordWithDB(hash, normdOrigSQL, db); record != nil { 
    // the old way
    return record
}
return h.bindInfo.Load().(cache).getBindRecord(hash, normdOrigSQL, db) // the new way
```
This implementation makes it more easy-to-use since then callers can avoid trying two ways by hand:
```
var record
if record = h. GetBindRecord(..., true); record != nil {
    ...
} else {
    record = h.GetBindRecord(..., false)
}
```

Reply:
Doing so will cause the old bindings to have a high priority, which is unreasonable. Because the new bindings cannot overwrite the old ones, this will make the user’s modification of bindings ineffective

Reply:
Oh, I got it wrong, I understand what you mean now.